scripts/ci-install-macos-latest.sh

```diff
@@ -1,4 +1,5 @@
 brew update
+brew install llvm --HEAD
```

Comment:
this might not be necessary... It was just and intend to use a newer clang (maybe necessary to use clang-XX). Since the built-in Apple clang is failing